pkg/investigations/chgm/chgm_hibernation_check.go

```diff
@@ -0,0 +1,87 @@
+package chgm
+
+import (
+	"sort"
+	"time"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	servicelogsv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
+	"github.com/openshift/configuration-anomaly-detection/pkg/ocm"
+)
+
+const RECENT_WAKEUP_TIME = 2 * time.Hour
+
+// 30 Days is always a problem as kubelet certificates will be expired
+const HIBERNATION_TOO_LONG = 30 * 24 * time.Hour
+const HIBERNATION_START_EVENT = "cluster_state_hibernating"
+const HIBERNATION_END_EVENT = "cluster_state_ready"
+const HIBERNATION_ONGOING_EVENT = "cluster_state_hibernating"
+const HIBERNATION_RESUME_EVENT = "cluster_state_resuming"
```

Comment:
**Question (non-blocking):** would it make sense to keep those consts private to the package instead?

Reply:
Ah yes - I guess `UPCASE` for all constants, is just something I have a hard time not doing, due to all the other languages, but I can put those private.